### PR TITLE
[WOR-1312] Azure e2e test of creating and deleting a workspace with WDS running

### PR DESCRIPTION
**Pull request by cahrens**
Ticket: https://broadworkbench.atlassian.net/browse/WOR-1322

Successful test run: https://github.com/broadinstitute/rawls/actions/runs/6671336059/job/18133849905

image

image

---

**PR checklist**

- [ ] Include the JIRA issue number in the PR description and title
- [ ] Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [ ] If you changed anything in `model/`, then you should [publish a new official `rawls-model`](https://github.com/broadinstitute/rawls/blob/develop/README.md#publish-rawls-model) and update `rawls-model` in [Orchestration's dependencies](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/project/Dependencies.scala).
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green, including CI tests
- [ ] **Squash commits and merge** to develop (branches are automatically deleted after merging)
- [ ] Inform other teams of any substantial changes via Slack and/or email

### CODE REVIEW

File: automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsConfig.scala [hunk 1]
```diff
@@ -13,4 +13,11 @@ object RawlsConfig extends CommonConfig {
   } else {
     "ERROR: workspaceManagerApiUrl unspecified! "
   }
+
+  // This value is only available when tests are run through the rawls-swat Github action (in a bee).
+  val leoUrl = if (fireCloud.hasPath("leonardoApiUrl")) {
```
Comment (cahrens) [inline automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsConfig.scala:18]: Was added in https://github.com/broadinstitute/terra-github-workflows/pull/131

File: automation/src/test/scala/org/broadinstitute/dsde/test/api/WorkspacesAzureApiSpec.scala [hunk 2]
```diff
@@ -57,7 +58,25 @@ class WorkspacesAzureApiSpec extends AnyFlatSpec with Matchers with BeforeAndAft
     }
   }
 
-  "Rawls" should "allow creation and deletion of azure workspaces" in {
+  "Other Terra services" should "include Leonardo" in {
```
Comment (cahrens) [inline automation/src/test/scala/org/broadinstitute/dsde/test/api/WorkspacesAzureApiSpec.scala:61]: Moved service check tests up to the top so it's easier to tell if we don't have access to a service.

**Comment by cahrens** [comment]
jenkins retest

**Review by blakery — APPROVED** [review]

**Comment by cahrens** [comment]
jenkins retest

**Review by aherbst-broad — APPROVED** [review]

**Pull request merged**